config/boards/radxa-e25.wip

```diff
@@ -0,0 +1,13 @@
+# Rockchip RK3568 quad core 1-8GB SoC GBe eMMC USB3
+BOARD_NAME="Radxa E25"
+BOARDFAMILY="rk35xx"
+BOOTCONFIG="radxa-e25-rk3568_defconfig"
+KERNEL_TARGET="legacy,edge"
```

Comment:
If we only have 6.1, then:
KERNEL_TARGET="edge"

Reply:
Hi Igor, 

short update to this, Legacy wortks fine with radxa kernel and uboot 4.19 from radxa but edge does not boot. i need to investigate more.

best regards